chain-abci/src/app/app_init.rs

```diff
@@ -300,6 +290,22 @@ impl<T: EnclaveProxy> ChainNodeApp<T> {
             let data = last_app_state.to_vec();
             let last_state =
                 ChainNodeState::decode(&mut data.as_slice()).expect("deserialize app state");
+            // TODO: genesis app hash check when embedded in enclave binary
+            let enclave_sanity_check = tx_validator.process_request(EnclaveRequest::CheckChain {
+                chain_hex_id,
```

Comment:
how about network_hex_id?

Reply:
a good point -- I think this naming inconsistency is in more places in abci, enclave-protocol etc., so I opened CRO-329 for fixing it